licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/decision/MandatoryCheckSpec.groovy

```diff
@@ -0,0 +1,58 @@
+package uk.gov.justice.digital.hmpps.licences.specs.decision
+
+import geb.spock.GebReportingSpec
+import spock.lang.Shared
+import spock.lang.Stepwise
+import uk.gov.justice.digital.hmpps.licences.pages.TaskListPage
+import uk.gov.justice.digital.hmpps.licences.pages.decision.ApprovalMandatoryCheckPage
+import uk.gov.justice.digital.hmpps.licences.pages.decision.ApprovalReleasePage
+import uk.gov.justice.digital.hmpps.licences.util.Actions
+import uk.gov.justice.digital.hmpps.licences.util.TestData
+
+@Stepwise
+class MandatoryCheckSpec extends GebReportingSpec {
+
+  @Shared
+  TestData testData = new TestData()
+  @Shared
+  Actions actions = new Actions()
+
+  def setupSpec() {
+    testData.loadLicence('decision/unstarted')
+    actions.logIn('DM')
+  }
+
+  def cleanupSpec() {
+    actions.logOut()
+  }
+
+  def 'Incomplete mandatory checks - continue journey'() {
```

Comment:
Integration tests to check the mandatory check page journeys. Tests Pass locally

Reply:
Should there be a test for not showing the check page if mandatory checks have been completed?